resources/en-US/capsule-info.bxb

```diff
@@ -0,0 +1,15 @@
+capsule-info {
+  dispatch-name (dice)
+  search-keywords{
+    keyword (dice)
+    keyword (die)
+    keyword (game)
+  }  
+  display-name (Dice Capsule)
+  developer-name (Bixby)
+  image-asset (/images/dice-icon.png)
```

Comment:
I would delete image-asset. It's not in use at the moment (marked as beta in the docs) and historically was a bigger different asset from the icon

Reply:
Thanks. Done. 👍